### PR TITLE
Fix multiple links rendering, sanitize HTML

### PR DESCRIPTION
**Pull request by antonmoiseev**
Related app [PR](https://github.com/Meet-Perry/perry/pull/685)

### CODE REVIEW

**Review by beastburn — COMMENTED** [review]
Is doing this via regex an XSS / javascript injection risk?

File: src/core/components/LinkifyMarkdown/index.js [hunk 1]
```diff
@@ -1,10 +1,11 @@
 import React from 'react';
+import sanitize from 'sanitize-html';
 
-const MARKDOWN_LINK_REGEX = /\[([^\\[]+)\]\((.*)\)/gm;
+const MARKDOWN_LINK_REGEX = /\[([^\\[]+)\]\(?(\S*)?\)/gm;
```
Comment (grandead) [inline src/core/components/LinkifyMarkdown/index.js:4]: image

Comment (grandead) [reply src/core/components/LinkifyMarkdown/index.js:4]: Works fine with common cases, is not foolproof in some cases
https://regex101.com/r/4y1VSg/1

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@meetperry/amity-uikit",
-  "version": "1.0.11-beta.3",
+  "version": "1.0.11-beta.4",
```
Comment (beastburn) [inline package.json:3]: do we need to bump the version in the existing app PR?

Comment (antonmoiseev) [reply package.json:3]: No, it is already beta.4

**Review by musatov — APPROVED** [review]

**Pull request merged**